scripts/maintenance/generate_audit_summary.py

```
@@ -0,0 +1,93 @@
+        for key, stat in MAPPING.items():
+            if key in issues_str:
+                stats[stat] += 1
+
+        branch = item['branch']
+        if branch in pr_map:
+            comment_on_pr(pr_map[branch], issues)
+    return stats
+
+def generate_markdown_summary(total, stats):
+    """Generates the Markdown daily summary string."""
+    date_str = datetime.now().strftime('%Y-%m-%d')
```

Ruff (0.15.13)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)